udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -37,484 +38,559 @@
+            </p>
+            {% if dataset.license %}
+            <h2 class="subtitle fr-mt-3v fr-mb-1v">{{ _('License') }}</h2>
+            <p class="fr-text--sm fr-mt-0 fr-mb-3v">
+                <code class="bg-alt-grey fr-px-1v text-grey-380">
+                {% if dataset.license.url %}<a href="{{ dataset.license.url }}">{% endif %}
+                {{ dataset.license.title }}
+                {% if dataset.license.url %}</a>{% endif %}
+                </code>
+            </p>
+            {% endif %}
+            {{quality_score_with_warning_below(dataset)}}
+        </section>
     </div>
-    {% endif %}
-    <div class="fr-grid-row fr-grid-row--gutters fr-mt-4w">
-        <div class="fr-col-12 fr-col-md-8">
-            <div class="fr-tabs">
-                <ul class="fr-tabs__list" role="tablist" aria-label="{{ _('Dataset informations') }}">
-                    <li role="presentation">
-                        <button id="tab-producer" class="fr-tabs__tab"
-                                tabindex="0" role="tab" aria-selected="true" aria-controls="tab-producer-panel">
-                            {% if dataset.organization %}
-                                {{ _('Producer') }}
-                            {% elif dataset.owner %}
-                                {{ _('Author') }}
-                            {% endif %}
-                        </button>
-                    </li>
-                    <li role="presentation">
-                        <button id="tab-metadata" class="fr-tabs__tab"
-                                tabindex="-1" role="tab" aria-selected="false" aria-controls="tab-metadata-panel">
-                            {{ _('Metadata') }}
-                        </button>
-                    </li>
-                    <li role="presentation">
-                        <button id="tab-actions" class="fr-tabs__tab"
-                                tabindex="-1" role="tab" aria-selected="false" aria-controls="tab-actions-panel">
-                            {{ _('Actions') }}
-                        </button>
-                    </li>
-                </ul>
-                <div
-                    id="tab-producer-panel"
-                    class="fr-tabs__panel fr-tabs__panel--selected"
-                    role="tabpanel"
-                    aria-labelledby="tab-producer"
+</div>
+<div class="fr-mb-10v">
+    {{ hook('dataset.display.after-description') }}
+</div>
+<div class="fr-tabs fr-tabs--full-page">
+    <div class="fr-container">
+        <ul class="fr-tabs__list" role="tablist" aria-label="{{ _('In-page navigation') }}">
+            <li role="presentation">
+                {% set count_resources = dataset.resources | length %}
+                <button
+                    id="resources"
+                    class="fr-tabs__tab"
                     tabindex="0"
+                    role="tab"
+                    {% if is_current_tab(request, "reuses_page") %}
```

Comment:
The DSFR component selects the tab with the `aria-selected` attribute but doesn't handle any navigation logic.

We do it with hash on the client side and trigger a tab change but this seems odd with server-side pagination. 

Without this, when you're on the reuses tabs, you click on the "next page" button, the page reload, you're on the "files" tab and then you're moved to the reuse tab again.

`reuses_page` is the only query parameter we have on the server side, because that's the only server side pagination left, that's why it's used here. Hash isn't sent so we can't use it in the template.